torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -2800,6 +2823,11 @@ def test_all_gather_full_group(self):
             group, group_id, rank = self._init_full_group_test()
             self._test_all_gather_helper(group, group_id, rank)
 
+        @unittest.skipIf(BACKEND == "nccl", "Nccl does not support CPU tensors")
```

Comment:
you probably need to import `unittest` package in this file to get rid of the CI failures :)